zerver/data_import/rocketchat.py

```diff
@@ -644,7 +644,9 @@ def list_reactions(reactions: Dict[str, Dict[str, Any]]) -> List[Dict[str, Any]]
     def message_to_dict(message: Dict[str, Any]) -> Dict[str, Any]:
         rc_sender_id = message["u"]["_id"]
         sender_id = user_id_mapper.get(rc_sender_id)
-        content = message["msg"]
+        content = ''
+        if 'msg' in message:
+            content = message["msg"]
```

Comment:
Zulip generally has an invariant that messages have non-empty content; so it's possible that it'd be better to skip such messages. 

(Though even better would be to print the attributes and debug to figure out what's going on -- maybe it's a special message type or something).

Reply:
I've changed it to `This message had no message content that could be imported from Rocket.Chat` and added a log statement.